server.js

```diff
@@ -0,0 +1,14 @@
+var express = require('express');
+var app = express();
+var server = require('http').createServer(app);
+
+app.use(express.static(__dirname + '/public'));
+app.use('/assets',  express.static(__dirname + '/bower_components'));
```

Comment:
I would probably specify ONE public folder and have everything served from there - but this is ok too